uws/UWS/client.py

```diff
@@ -50,15 +51,40 @@ def _validate_and_parse_filters(self, filters):
             if phase not in models.JobPhases.phases:
                 raise UWSError("Unknown phase %s in filter", phase)
 
-        params = [("PHASE",phase) for phase in phases]
+        params = [("PHASE", phase) for phase in phases]
 
         return params
 
-    def get_job(self, id):
+    def _validate_and_parse_wait(self, wait, phase=None):
         try:
-            response = self.connection.get(id)
-        except Exception as e:
-            raise UWSError(str(e))
+            duration = int(wait)
```

Comment:
you could also use `wait.isnumeric()` here, and check for floats (if string includes a `.`). Because int casting can lead to undesired effects. However I'm also ok with this approach here...